cmd/run.go

```diff
@@ -88,6 +90,14 @@ A full workflow is capable of running the following steps:
 			Name:        "skip-compile",
 			Description: "skip compilation when generating the SDK",
 		},
+		flag.BooleanFlag{
+			Name:        "skip-compile",
```

Comment:
Oops. We currently have a check that makes sure all flags defined here have a corresponding `json` tag in the flag type struct. We could probably extend that to do the reverse check as well, to avoid this problem

Reply:
I'm confused, didn't I add that?

Reply:
https://github.com/speakeasy-api/speakeasy/blob/41256e756922e997f74a5b02ff3d5332d423aa36/cmd/run.go#L30

Reply:
oh lol I see it